dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/infer/InferrerHelper.java

```diff
@@ -125,6 +125,13 @@ protected boolean isDescriptorPath(String path) {
                     return path.endsWith(".ga");
                 }
             };
+        case SMK:
+            return new DescriptorLanguageInferrer(DescriptorLanguage.SMK) {
+                @Override
+                protected boolean isDescriptorPath(String path) {
+                    return path.endsWith("Snakefile");
```

Comment:
This'll match any file name that ends with `Snakefile`, like `mySnakefile` or `some.Snakefile`.  Are those valid Snakefile names?  If not, add a leading slash?